lonestar/analytics/distributed/README.md

```diff
@@ -114,22 +122,74 @@ Heterogeneous apps have additional command line parameters:
 6 processes across the 2 machines (1 for each GPU).
 
-Also, it suffices to use only one "c" in pset to run on CPUs on your machines: 
-you can specify the amount of CPUs to use using the aforementioned thread 
+Also, it suffices to use only one "c" in pset to run on CPUs on your machines:
+you can specify the amount of CPUs to use using the aforementioned thread
 option `-t`.
 
+Performance Considerations
+================================================================================
+
+* As mentioned above if running multiple processes on a single machine,
+  specifying `GALOIS_DO_NOT_BIND_THREADS=1` as an environment variable is
+  crucial for performance.
+
+* We have also observed that `GALOIS_DO_NOT_BIND_THREADS=1` to improve
+  performance in a distributed setting as well (multiple hosts each with its
+  own process).
+
+* For 16 or less hosts/GPUs, for performance, we recommend using an
+  **edge-cut** partitioning policy (OEC or IEC) with **synchronous**
+  communication for performance.
+
+* For 32 or more hosts/GPUs, for performance, we recommend using the
+  **Cartesian vertex-cut** partitioning policy (CVC) with **asynchronous**
+  communication for performance.
+
+Publications Related to Distributed Applications
+================================================================================
+
+Please see the publications listed below for information on the distributed
+runtime as well as performance studies we have conducted over the years.
+
+Roshan Dathathri, Gurbinder Gill, Loc Hoang, Hoang-Vu Dang, Alex Brooks,
+Nikoli Dryden, Marc Snir, Keshav Pingali, “Gluon: A Communication-Optimizing
+Substrate for Distributed Heterogeneous Graph Analytics,” Proceedings of the
+39th ACM SIGPLAN Conference on Programming Language Design and Implementation
+(PLDI), June 2018.
+
+Gurbinder Gill, Roshan Dathathri, Loc Hoang, Keshav Pingali, “A Study of
+Partitioning Policies for Graph Analytics on Large-scale Distributed
+Platforms,” Proceedings of the 45th International Conference on Very Large Data
+Bases (PVLDB), 12(4): 321-334, December 2018.
+
+Loc Hoang, Roshan Dathathri, Gurbinder Gill, Keshav Pingali, “CuSP: A
+Customizable Streaming Edge Partitioner for Distributed Graph Analytics,”
+Proceedings of the 33rd IEEE International Parallel and Distributed Processing
+Symposium (IPDPS), May 2019.
+
+Roshan Dathathri, Gurbinder Gill, Loc Hoang, Hoang-Vu Dang, Vishwesh Jatala, V.
+Krishna Nandivada, Marc Snir, Keshav Pingali, “Gluon-Async: A Bulk-Asynchronous
+System for Distributed and Heterogeneous Graph Analytics,” Proceedings of the
+28th IEEE International Conference on Parallel Architectures and Compilation
+Techniques (PACT), September 2019.
+
+Vishwesh Jatala, Roshan Dathathri, Gurbinder Gill, Loc Hoang, V. Krishna
+Nandivada, Keshav Pingali, “A Study of Graph Analytics for Massive Datasets on
+Distributed GPUs,” Proceedings of the 34th IEEE International Parallel and
+Distributed Processing Symposium (IPDPS), May 2020.
+
```

Comment:
You should add our other papers as well. Abelian, MRBC, Phoenix, etc.

Reply:
Abelian/MRBC sure. Phoenix, maybe not because it's not available anywhere in our code at the moment. Actually, even Abelian falls in the same bucket. Thoughts?

Reply:
Yes, it's just FYI.